core/libp2p/peer/peer_info.hpp

```diff
@@ -9,13 +9,13 @@
 #include "libp2p/multi/multiaddress.hpp"
 #include "libp2p/multi/multihash.hpp"
 
-namespace libp2p::common {
+namespace libp2p::peer {
   /**
    * Information about the peer in the network
    */
   struct PeerInfo {
-    multi::Multihash peer_id;
-    multi::Multiaddress peer_address;
+//    multi::Multihash peer_id;
+//    multi::Multiaddress peer_address;
```

Comment:
?

Reply:
This class is going to be implemented in the next PR, so for now it's just left empty